From 27bf0e2901e2281143f144324b151a40654f6a5b Mon Sep 17 00:00:00 2001
From: Richard Hughes <richard@hughsie.com>
Date: Thu, 29 Sep 2011 14:32:17 +0100
Subject: [PATCH 2/8] power: Do not revert to the pre-idle brightness if idle
 dimming is disabled

We want to set manager->priv->pre_dim_brightness = -1 to indicate that we're
restored the value and that no further restoration is required, as the user may
have already altered the level.

This will only happen if you disable dimming at runtime after having dimmed at
least once.

Resolves https://bugzilla.gnome.org/show_bug.cgi?id=660434
---
 plugins/power/gsd-power-manager.c |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/plugins/power/gsd-power-manager.c b/plugins/power/gsd-power-manager.c
index 370d90d..a1bc213 100644
--- a/plugins/power/gsd-power-manager.c
+++ b/plugins/power/gsd-power-manager.c
@@ -2756,6 +2756,7 @@ idle_set_mode (GsdPowerManager *manager, GsdPowerIdleMode mode)
                                 g_error_free (error);
                                 return;
                         }
+                        manager->priv->pre_dim_brightness = -1;
                 }
         }
 }
-- 
1.7.7

